alphalens/performance.py

```diff
@@ -400,7 +400,7 @@ def quantile_turnover(quantile_factor, quantile):
     return quant_turnover
 
 
-def factor_rank_autocorrelation(factor, time_rule='W', by_group=False):
+def factor_rank_autocorrelation(factor, period=1, time_rule=None, by_group=False):
```

Comment:
A default like ''W' makes sense for daily data but as I understood that alphalens is frequency agnostic so i figured out there isn't a default that makes sense all the time (and default wasn't used anyway).
An alternative option that works at all data frequencies would be to calculate the factor mean value along the period with rolling_mean instead of using the time_rule. I still don't like it by default though. 

Those are just my thoughts anyway. More than happy to revert back the code as it was.